docs/conf.py

```diff
@@ -62,17 +62,17 @@
 
 # General information about the project.
 project = 'microraiden'
-copyright = '2017, Brainbot AG'
-author = 'Brainbot AG'
+copyright = '2018, brainbot labs Est.'
+author = 'brainbot labs Est.'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '0.1'
+version = '0.2'
 # The full version, including alpha/beta/rc tags.
-release = '0.1.0'
+release = '0.2.0'
```

Comment:
can't we simply extract the version from `microraiden/VERSION` ?

Reply:
Sure, but I did not want to touch this too much, knowing that @ezdac is currently working on the docs. I just wanted to get rid of the legal issues asap.

Reply:
Extraction is implemented in the `docs-develop` branch.
This hotfix won't be enough for RTD to build, since it currently is missing a `docs/requirements.txt` file. I'll try to do free some time and do a PR this evening that this can be resolved asap.

Reply:
Ok, version is not that important for the scope of this PR, so I'm merging it.